<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Layout_checkClass</class>
 <widget class="QMainWindow" name="Layout_checkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layout_check</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Layout_check.qrc"/>
 </resources>
 <connections/>
</ui>
